foundations/javascript_basics/understanding_errors.md

```diff
@@ -0,0 +1,132 @@
+# Introduction
+
+Reading error messages is an underrated art form. At first glance, many beginners shrink away from error messages as they appear to be “scary” and difficult to understand because they include terms one may not be familiar with. However, error messages provide developers with a treasure trove of knowledge, and tell you everything you need to know about how to resolve them! Being able to parse error messages and warnings without fear will enable you to be able to effectively debug your applications, receive meaningful help from others, and empower yourself to push forward when faced with an error.
```

Comment:
I would maybe not say "art form" as it's not something that you can do without. It's an absolute requirement to being a developer.